src/main/java/core/basesyntax/services/impl/DataConverterServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.services.DataConverterService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterServiceImpl implements DataConverterService {
+    private static final int FRUIT_OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int FRUIT_QUANTITY_INDEX = 2;
+    private static final int DATA_INDEX = 1;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> parsedLines) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (int i = DATA_INDEX; i < parsedLines.size(); i++) {
```

Comment:
The variable `DATA_INDEX` is used to skip the first line of the input assuming it's a header. However, this is not clear from the name of the variable. Consider renaming it to `HEADER_INDEX` or adding a comment explaining its purpose.